news/1605.feature

```diff
@@ -0,0 +1 @@
+Add UID to relationvalue_converter summary. @ksuess
```

Comment:
@ksuess we use [ksuess] in backend packages. :) Just FYI. I will fix this for you.

Reply:
@tisto We just decided to switch to using `@githubusername` everywhere, for consistency. https://github.com/plone/documentation/pull/1478#discussion_r1157184539

Reply:
As a bonus, GitHub renders `@username` format to a clickable username.

Reply:
@davisagli ok, that's new for me. Awesome! That makes lots of sense to me. Will stick to this from now on for plone.rest(api) releases.

Reply:
@ksuess sorry, for the noise. Turns out you were right and I was wrong. :)